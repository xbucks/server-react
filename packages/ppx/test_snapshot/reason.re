let%browser_only loadInitialText = () => {
  setHtmlFetchState(Loading);
  WcApi.fetchHTML(WcConstants.initialText)
  ->Promise.flatMap(html => onChange(html->WcStringHelpers.htmlToText)->Promise.resolved)
  ->Promise.Js.catch(_ => setHtmlFetchState(Error)->Promise.resolved)
  ->ignore;
};
