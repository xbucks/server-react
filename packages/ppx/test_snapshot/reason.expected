[@warning "-27"]
let loadInitialText = () =>
  raise(
    ReactDOM.Impossible_in_ssr(
      "fun () ->\n  setHtmlFetchState Loading;\n  (((((WcApi.fetchHTML WcConstants.initialText) |. Promise.flatMap)\n       (fun html ->\n          (onChange (html |. WcStringHelpers.htmlToText)) |. Promise.resolved))\n      |. Promise.Js.catch)\n     (fun _ -> (setHtmlFetchState Error) |. Promise.resolved))\n    |. ignore",
    ),
  );
